uxkit-youtube/src/main/java/com/voxeet/uxkit/youtube/YoutubeMediaPresentationView.java

```diff
@@ -98,43 +133,87 @@ public void pause(@NonNull VideoPresentationPaused videoPresentationPaused) {
+        }
     }
 
     /**
      * Callback when youtube has been properly initialized
      *
-     * @param provider      the specific provider for the session
      * @param youTubePlayer the player to use
-     * @param b
      */
-    @Override
-    public void onInitializationSuccess(@NonNull YouTubePlayer.Provider provider, @NonNull YouTubePlayer youTubePlayer, boolean b) {
+    private void onInitializationSuccess(@NonNull YouTubePlayer youTubePlayer) {
+        Log.d("onInitializationSuccess " + youtubePlayer);
         this.youtubePlayer = youTubePlayer;
-    }
+        youtubeView.setEnabled(false);
+        youtubeView.setOnTouchListener((view1, motionEvent) -> false);
 
-    /**
-     * Played to initialize the provider and the player
-     *
-     * @param provider                    the specific provider
-     * @param youTubeInitializationResult the initialization result with the reason
-     */
-    @Override
-    public void onInitializationFailure(@NonNull YouTubePlayer.Provider provider, @NonNull YouTubeInitializationResult youTubeInitializationResult) {
+        youtubeView.addFullScreenListener(new YouTubePlayerFullScreenListener() {
+            @Override
+            public void onYouTubePlayerEnterFullScreen() {
+                Log.d("onYouTubePlayerEnterFullScreen");
+            }
 
+            @Override
+            public void onYouTubePlayerExitFullScreen() {
+                Log.d("onYouTubePlayerEnterFullScreen");
+            }
+        });
+
+        youtubeView.enableBackgroundPlayback(true);
+
+        for (Event pending : pendingEvents) {
+            switch (pending.getClass().getSimpleName()) {
+                case "VideoPresentationStarted":
+                    start((VideoPresentationStarted) pending);
+                    break;
+                case "VideoPresentationStopped":
+                    stop((VideoPresentationStopped) pending);
+                    break;
+                case "VideoPresentationPlay":
+                    play((VideoPresentationPlay) pending);
+                    break;
+                case "VideoPresentationPaused":
+                    pause((VideoPresentationPaused) pending);
+                    break;
+                case "VideoPresentationSeek":
+                    seek((VideoPresentationSeek) pending);
+                    break;
+                default:
+                    Log.d("unknown class " + pending.getClass().getSimpleName());
+            }
+        }
+
+        pendingEvents.clear();
     }
 
-    private void init(@NonNull String youtubeKey) {
+    private void init() {
         try {
-            View view = LayoutInflater.from(getContext()).inflate(R.layout.voxeet_youtube, this, false);
-            addView(view);
-            FrameLayout parent = findViewById(R.id.voxeet_youtube_parent);
-            youtubeView = ViewProvider.createView(getContext());
+            LayoutInflater.from(getContext()).inflate(R.layout.voxeet_youtube, this);
```

Comment:
It's actually mandatory, I forgot to mention the tip status. It will change because the view will be moved to be a child directly so that the container will contain also another component to manage the media volume (since this library is using a player on this mode whereas we are focusing onto call mode -> vol +/vol - can't change YouTube's sound volume